### PR TITLE
Drupal book page

### PR DESCRIPTION
**Pull request by miri**
Squashed commit of #45

### CODE REVIEW

File: haskala/modules/custom/haskala_book/plugins/content_types/book_fields/book_fields.inc [hunk 1]
```diff
@@ -0,0 +1,103 @@
+<?php
+
+$plugin = array(
+  'title' => t('Book fields'),
+//  'description' => t('Add a node from your site as content.'),
```
Comment (bitamar) [inline haskala/modules/custom/haskala_book/plugins/content_types/book_fields/book_fields.inc]: => 'description' => t('Display a book.'),

File: haskala/modules/custom/haskala_book/plugins/content_types/book_fields/book_fields.inc [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  'category' => t('Haskala'),
+  'hook theme' => 'haskala_book_book_fields_content_type_theme',
+  'required context' => new ctools_context_required(t('Node'), 'node'),
+);
+
+/**
+ * Output function for the 'node' content type.
```
Comment (bitamar) [inline haskala/modules/custom/haskala_book/plugins/content_types/book_fields/book_fields.inc]: Change comment to: "Render callback."

File: haskala/modules/custom/haskala_book/plugins/content_types/book_fields/book_fields.inc [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  }
+
+  $node = $context->data;
+  $wrapper = entity_metadata_wrapper('node', $node);
+
+  $top_details_group = field_group_load_field_group('group_top_details', 'node', 'book', 'default');
+  $top_details_group_fields = array();
+  foreach($top_details_group->children as $field_name) {
+
+    // Check if the field isn't empty
```
Comment (bitamar) [inline haskala/modules/custom/haskala_book/plugins/content_types/book_fields/book_fields.inc]: Missing .

File: haskala/modules/custom/haskala_book/plugins/content_types/book_fields/book_fields.inc [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+ */
+function haskala_book_book_fields_content_type_render($subtype, $conf, $args, $context) {
+  if (!empty($context) && empty($context->data)) {
+    return;
+  }
+
+  $node = $context->data;
+  $wrapper = entity_metadata_wrapper('node', $node);
+
+  $top_details_group = field_group_load_field_group('group_top_details', 'node', 'book', 'default');
```
Comment (bitamar) [inline haskala/modules/custom/haskala_book/plugins/content_types/book_fields/book_fields.inc]: Add comment

File: haskala/modules/custom/haskala_book/plugins/content_types/book_fields/book_fields.inc [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+  $variables = array(
+    'nid' => $node->nid,
+    'grouped_fields' => theme('haskala_book_fields', $grouped_fileds_variables),
+  );
+
+
+
+  theme('parent_group', $parent_group);
+  theme('groups', $field_groups);
```
Comment (bitamar) [inline haskala/modules/custom/haskala_book/plugins/content_types/book_fields/book_fields.inc]: Remove

File: haskala/modules/custom/haskala_book/plugins/content_types/book_fields/haskala-book-fields.tpl.php [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+ */
+?>
+
+      <section class="top-details">
+        <?php if (!empty($title)): ?>
+          <h1 class="title page-header"><?php print $title; ?></h1>
+        <?php endif; ?>
+
+          <?php
+          //print render($content);
```
Comment (bitamar) [inline haskala/modules/custom/haskala_book/plugins/content_types/book_fields/haskala-book-fields.tpl.php]: remove

File: haskala/modules/custom/haskala_book/plugins/content_types/book_fields/haskala-book.tpl.php [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+ *
+ * Available variables:
+ * - $nid: the nid of the node.
+ * - $grouped_fields: render haskala-book-fields.tpl
+ *
+ */
+?>
+
+<section>
+  <h3 class="hr-title">BOOK</h3>
```
Comment (bitamar) [inline haskala/modules/custom/haskala_book/plugins/content_types/book_fields/haskala-book.tpl.php]: Texts should be in normal case. Convert it to uppercase with css

File: haskala/modules/custom/haskala_book/plugins/content_types/book_fields/haskala-book.tpl.php [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+  <div class="main-content">
+    <div id="node-<?php print $nid; ?>" class="node node-book clearfix">
+          <?php print $grouped_fields; ?>
```
Comment (bitamar) [inline haskala/modules/custom/haskala_book/plugins/content_types/book_fields/haskala-book.tpl.php]: Fix indentation

File: haskala/modules/custom/haskala_book/plugins/content_types/book_fields/book_fields.inc [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+ */
+function haskala_book_book_fields_content_type_render($subtype, $conf, $args, $context) {
+  if (!empty($context) && empty($context->data)) {
+    return;
+  }
+
+  $node = $context->data;
+  $wrapper = entity_metadata_wrapper('node', $node);
+
+  //Fields to be displayed above the book tabs.
```
Comment (bitamar) [inline haskala/modules/custom/haskala_book/plugins/content_types/book_fields/book_fields.inc]: Make sure all comments start with a space.

File: haskala/modules/custom/haskala_book/plugins/content_types/book_fields/haskala-book-fields.tpl.php [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+ *
+ * Available variables:
+ * - $title: the (sanitized) title of the node.
+ * - $top_details_group: An array of node book top details group fields items.
+ * - $tabs: An array of node book bottom details groups fields items.
+ *
+ */
+?>
+
+      <section class="top-details">
```
Comment (bitamar) [inline haskala/modules/custom/haskala_book/plugins/content_types/book_fields/haskala-book-fields.tpl.php]: Fix indentation

File: jekyll/app/_includes/footer.html [hunk 11]
```diff
@@ -7,12 +7,14 @@
...
-      <li class="first last login"><a href="#">LOGIN</a></li>
-    </ul>
-  </nav>
+  <div class="login-area">
+    <nav>
+      <ul>
+        <li class="first"><a href="#">Add a book</a></li>
+        <li><a href="#">Add a book series</a></li>
+        <li class="last"><a href="#">Add a person</a></li>
+        <!--<li class="last login"><a href="#">LOGIN</a></li>-->
```
Comment (bitamar) [inline jekyll/app/_includes/footer.html]: Remove first and last classes (Use first-child instead). 
LOGIN => Login

File: behat/features/person.feature [hunk 12]
```diff
@@ -3,7 +3,7 @@ Feature: Person
   As an anonymous user
   We need to be able to have access to a person page
 
-  @api
+  @api @wip
```
Comment (bitamar) [inline behat/features/person.feature]: remove wip

File: behat/features/edition.feature [hunk 13]
```diff
@@ -3,7 +3,7 @@ Feature: Edition
   As an anonymous user
   We need to be able to have access to a edition page
 
-  @api
+  @api @wip
```
Comment (bitamar) [inline behat/features/edition.feature]: remove wip

File: haskala/modules/custom/haskala_general/haskala_general.features.menu_links.inc [hunk 14]
```diff
@@ -90,61 +90,61 @@ function haskala_general_menu_default_menu_links() {
     'weight' => 0,
     'customized' => 1,
   );
-  // Exported menu link: menu-icons-menu_digital-books:<front>
-  $menu_links['menu-icons-menu_digital-books:<front>'] = array(
+  // Exported menu link: menu-icons-menu_digital-books:node/2592
```
Comment (bitamar) [inline haskala/modules/custom/haskala_general/haskala_general.features.menu_links.inc]: this seems wrong - why do we mention this node id?

File: haskala/modules/custom/haskala_general/haskala_general.info [hunk 15]
```diff
@@ -16,13 +16,16 @@ features[menu_links][] = menu-bottom-menu_add-a-book-series:<front>
 features[menu_links][] = menu-bottom-menu_add-a-book:<front>
 features[menu_links][] = menu-bottom-menu_add-a-person:<front>
 features[menu_links][] = menu-icons-menu_books:<front>
-features[menu_links][] = menu-icons-menu_digital-books:<front>
-features[menu_links][] = menu-icons-menu_people:<front>
-features[menu_links][] = menu-icons-menu_places:<front>
+features[menu_links][] = menu-icons-menu_digital-books:node/2592
```
Comment (bitamar) [inline haskala/modules/custom/haskala_general/haskala_general.info]: Probably needs to be reverted to <front>

File: jekyll/app/book.html [hunk 16]
```diff
@@ -1,44 +1,44 @@
 ---
 layout: default
 title: Haskala
-body_class: page-single-book
+body_class: node-type-book
 ---
 <section>
-  <h3 class="hr-title">BOOK</h3>
+  <h3 class="hr-title">book</h3>
```
Comment (bitamar) [inline jekyll/app/book.html]: Should be "Book"

File: haskala/modules/custom/haskala_general/haskala_general.features.menu_links.inc (hunk 14 above)
Comment (miri) [inline haskala/modules/custom/haskala_general/haskala_general.features.menu_links.inc]: It's only for prevent link be active. that linkes's pages not exit yet.

**Comment by bitamar** [comment]
Merged

**Pull request closed without merging**